core/src/main/java/org/elasticsearch/common/network/InetAddresses.java

```diff
@@ -23,10 +25,12 @@
 import java.nio.ByteBuffer;
 import java.util.Arrays;
 import java.util.Locale;
+import java.util.regex.Pattern;
 
 public class InetAddresses {
     private static int IPV4_PART_COUNT = 4;
     private static int IPV6_PART_COUNT = 8;
+    private static final Pattern MASK_PATTERN = Pattern.compile("[\\.|/]");
```

Comment:
Sure, that was just the one way the range builder was doing it. I'll change it to use a Matcher and a stricter pattern.